src/adapter/src/explain/optimizer_trace.rs

```diff
@@ -52,7 +53,8 @@ impl OptimizerTrace {
     /// accumulate all [`TraceEntry`] instances otherwise.
     pub fn new(broken: bool, path: Option<&'static str>) -> OptimizerTrace {
         if broken {
-            let subscriber = DelegateSubscriber::default()
+            eprintln!("TRIGGERED");
```

Comment:
Is the `eprintln` needed here?

Reply:
no, this was just cause its a draft! i need to clean this pr up!